src/main/kotlin/uk/gov/justice/digital/hmpps/subjectaccessrequestworker/utils/TemplateHelpers.kt

```diff
@@ -59,4 +59,6 @@ class TemplateHelpers(
 
   fun buildDateNumber(year: Number?, month: Number?, day: Number?): String =
     buildDate(year?.toInt()?.toString(), month?.toInt()?.toString(), day?.toInt()?.toString())
+
+  fun eq(input: String?, value: String?): Boolean = input == value
```

Comment:
This only seems to be used in a test, is it still needed? What value does it add over the standard `==`?

Reply:
Oh, I see it in the mustache template now, sorry